agent/containers/images/Makefile

```diff
@@ -127,11 +127,11 @@ tm: pkgmgr-clean all-tags $(_DEFAULT_DISTROS:%=%-tool-meister-tagged)
 #-
 
 # We also offer targets per distribution target
-${_DISTROS}: %: pkgmgr-clean %-all-tagged
+${_DISTROS}: %: all-tags %-all-tagged
 
-$(_DISTROS:%=%-tds): %-tds: pkgmgr-clean %-tool-data-sink-tagged
+$(_DISTROS:%=%-tds): %-tds: all-tags %-tool-data-sink-tagged
 
-$(_DISTROS:%=%-tm): %-tm: pkgmgr-clean %-tool-meister-tagged
+$(_DISTROS:%=%-tm): %-tm: all-tags %-tool-meister-tagged
```

Comment:
This change is not a good idea:  it will prevent us from building a distro which is not in the default list.

These targets are intended to build for a single distro.  As such, they don't need (or want!) to depend on `all-tags`, because the single distro has no requirement to be consistent with any other distro.

This change should be reverted.

Reply:
> it will prevent us from building a distro which is not in the default list.

OK, it won't actually _prevent_ it -- what will happen is that `make` will go and generate the `*-tag.lis` files for each of the default distros, make sure that they all match each other, and then proceed to ignore them and generate the `*-tag.lis` file for the distro that it actually intends to build (and it won't compare it to the others).  Unfortunately, generating the `*-tag.lis` files is slow, so it's painful to be doing it for no purpose.

Moreover, if for some reason one of the default `*-tag.lis` files cannot be built, then the target won't be built either, even though it doesn't depend on the defaults' files.  (Obviously, this can be worked around by overriding the default distro list, but that shouldn't be have to be necessary.)